### ISSUE TITLE
undefined method `chomp' for nil:NilClass

### ISSUE THREAD

**Issue by aaronjensen** [castwide/solargraph#313]
I'm getting this with solargraph 0.39.0:

**edit**: also happens on 0.39.2, but not on 0.38.0

```
[WARN] Error processing request: [NoMethodError] undefined method `chomp' for nil:NilClass
[WARN] ~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:110:in `process_directive'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:72:in `block in process_comment'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:69:in `each'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:69:in `process_comment'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:190:in `block in process_comment_directives'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:187:in `each'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:187:in `process_comment_directives'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:27:in `map'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map/mapper.rb:50:in `map'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/source_map.rb:141:in `map'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/api_map.rb:88:in `block in catalog'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/api_map.rb:70:in `each'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/api_map.rb:70:in `catalog'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/library.rb:23:in `initialize'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/library.rb:363:in `new'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/library.rb:363:in `load'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/host.rb:278:in `prepare'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/host.rb:295:in `block in prepare_folders'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/host.rb:294:in `each'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/host.rb:294:in `prepare_folders'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/message/initialize.rb:14:in `block in process'
~/ruby/2.7.1/lib/ruby/2.7.0/benchmark.rb:293:in `measure'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/message/initialize.rb:10:in `process'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/host.rb:101:in `receive'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/transport/adapter.rb:44:in `process'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/transport/adapter.rb:17:in `block in opening'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/transport/data_reader.rb:59:in `parse_message_from_buffer'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/transport/data_reader.rb:35:in `block in receive'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/transport/data_reader.rb:30:in `each_char'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/transport/data_reader.rb:30:in `receive'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/transport/adapter.rb:27:in `receiving'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/client.rb:63:in `tick'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/server/stdio.rb:18:in `update'
~/ruby/2.7.1/lib/ruby/2.7.0/observer.rb:197:in `block in notify_observers'
~/ruby/2.7.1/lib/ruby/2.7.0/observer.rb:196:in `each'
~/ruby/2.7.1/lib/ruby/2.7.0/observer.rb:196:in `notify_observers'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/client.rb:121:in `read_input'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/client.rb:102:in `block in run_input_thread'
~/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.39.0/lib/solargraph/language_server/uri_helpers.rb:17: warning: URI.unescape is obsolete
```

**Comment by castwide** [comment]
Fixed in gem v0.39.3.

**Comment by aaronjensen** [comment]
Confirmed! Thank you for the quick fix.
